Maestro/subscriptions.json

```diff
@@ -933,7 +933,26 @@
         }
       }
     },
-    // Automate opening PRs to merge cli release/8.0.3xx to main
+    // Automate opening PRs to merge cli release/8.0.4xx to main
```

Comment:
```suggestion
    // Automate opening PRs to merge sdk repos from release/8.0.4xx to main
```

Reply:
Now this comment is different from all the others. If it doesn't bother you, I can leave it; I just normally like things to match.